packages/currency/src/loaders/initial-data.ts

```diff
@@ -0,0 +1,23 @@
+import { ModulesSdkTypes, LoaderOptions } from "@medusajs/types"
+import { defaultCurrencies } from "@medusajs/utils"
+import { Currency } from "@models"
+
+export default async ({
+  container,
+  options,
+}: LoaderOptions<
+  | ModulesSdkTypes.ModuleServiceInitializeOptions
+  | ModulesSdkTypes.ModuleServiceInitializeCustomDataLayerOptions
+>): Promise<void> => {
+  const {
```

Comment:
@adrien2p this works, but I'd like to hear how we can improve it if possible (eg. how to get the currencyModuleService name, etc.